reward/reward_distributor.go

```diff
@@ -43,14 +42,12 @@ func isEmptyAddress(addr common.Address) bool {
 }
 
 type RewardDistributor struct {
-	sm  *stakingManager
 	rcc *rewardConfigCache
 	gh  governanceHelper
 }
 
-func NewRewardDistributor(bc *blockchain.BlockChain, gh governanceHelper) *RewardDistributor {
+func NewRewardDistributor(gh governanceHelper) *RewardDistributor {
```

Comment:
Method description comment is missing.

Reply:
I'll make a new pr for method description comments